buildpacks/sbt/tests/integration_tests.rs

```diff
@@ -58,6 +60,17 @@ fn test_play_support_for_v2_7() {
     })
 }
 
+#[test]
+#[ignore = "integration test"]
+fn test_batch_mode_warning_suppression() {
```

Comment:
I want to restructure the tests in general. For this PR I just added a test like the others.